src/sentry/static/sentry/app/components/dropdownAutoCompleteMultiRow.jsx

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import CheckboxFancy from 'app/components/checkboxFancy';
+import space from 'app/styles/space';
+
+class DropdownAutoCompleteMultiRow extends React.Component {
+  static propTypes = {
+    checked: PropTypes.bool,
+    noMultiFeature: PropTypes.bool,
```

Comment:
nit: This is typically named `multi` in other components

Reply:
Alternatively, we can not use this component in the non-multi case?

Reply:
I'm kinda of angling here for the free-tier project selector to ultimately look exactly like the multi selector, minus the "multi" part. That way we won't accidentally regress the experience of free tier.

So my inclination is just to change this to 'multi' and keep going.

Reply:
Sounds good